docs/agent/changelog.md

```diff
@@ -6,6 +6,20 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.15.0 - \[2024-08-29\]
+
+- Added `--url` agent CLI flag for simpler creation of HTTP(S), TLS, and TCP protocol endpoints.
+- Deprecated `--domain`, `--scheme`, and `--remote-addr` agent CLI flags, which have been replaced by `--url`.
+- Added `--metadata` and `--description` agent CLI flags when using `--url`.
+
+##### A new agent configuration format added to V3
+
+- Added `agent:` as a toplevel field to agent config, with `endpoints:` as a nested field for easier [endpoint creation](/docs/agent/config/v3/#full-example).
+- v3 supports _both_ tunnels and endpoints. Tunnels are now considered deprecated when using config v3.
+- Some global configuration fields renamed in v3 when used under the `agent:` field.
```

Comment:
s/global/agent

The verbiage is agent configuration for anything that isn't "make tunnel go brr" - this is reflected by it's new location

Reply:
also no global configuration exists in version 3, it's all under agent

Reply:
The wording here kills me, cause it's _called_ `global` elsewhere:

<img width="408" alt="image" src="https://github.com/user-attachments/assets/fffb4a00-4a1a-492c-b167-701cceb9b76c">

Maybe I should just change it to `agent` all over like said : |

Reply:
Will update as part of my _other_ pr 🖖